src/Uno.UWPSyncGenerator.Reference/Uno.UWPSyncGenerator.Reference.csproj

```diff
@@ -1,212 +1,202 @@
-		<NoWarn>;2008</NoWarn>
-		<DebugType>full</DebugType>
-		<PlatformTarget>x64</PlatformTarget>
-		<UseVSHostingProcess>false</UseVSHostingProcess>
-		<ErrorReport>prompt</ErrorReport>
-	</PropertyGroup>
-	<PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
-		<PlatformTarget>x64</PlatformTarget>
-		<OutputPath>bin\x64\Release\</OutputPath>
-		<DefineConstants>TRACE;NETFX_CORE;WINDOWS_UWP</DefineConstants>
-		<Optimize>true</Optimize>
-		<NoWarn>;2008</NoWarn>
-		<DebugType>pdbonly</DebugType>
-		<PlatformTarget>x64</PlatformTarget>
-		<UseVSHostingProcess>false</UseVSHostingProcess>
-		<ErrorReport>prompt</ErrorReport>
-	</PropertyGroup>
-	<ItemGroup>
-		<!-- A reference to the entire .Net Framework and Windows SDK are automatically included -->
-	</ItemGroup>
-	<ItemGroup>
-		<Compile Include="Class1.cs" />
-		<Compile Include="Properties\AssemblyInfo.cs" />
-		<EmbeddedResource Include="Properties\Uno.UWPSyncGenerator.Reference.rd.xml" />
-	</ItemGroup>
-	<ItemGroup>
-		<PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-			<Version>6.2.9</Version>
-		</PackageReference>
-		<SDKReference Include="WindowsDesktop, Version=10.0.19041.0">
-			<Name>Windows Desktop Extensions for the UWP</Name>
-		</SDKReference>
-		<SDKReference Include="WindowsMobile, Version=10.0.19041.0">
-			<Name>Windows Mobile Extensions for the UWP</Name>
-		</SDKReference>
-	</ItemGroup>
-	<Choose>
-		<When Condition="'$(UNO_UWP_BUILD)'=='false'">
-			<ItemGroup>
-				<PackageReference Include="Microsoft.WindowsAppSDK">
-					<Version>1.0.0</Version>
-				</PackageReference>
-				<PackageReference Include="Microsoft.WindowsAppSDK.WinUI">
-					<Version>1.0.0-experimental1</Version>
-				</PackageReference>
-				<PackageReference Include="Microsoft.WindowsAppSDK.Foundation">
-					<Version>1.0.0-experimental1</Version>
-				</PackageReference>
-				<PackageReference Include="Microsoft.WindowsAppSDK.InteractiveExperiences">
-					<Version>1.0.0-experimental1</Version>
-				</PackageReference>	
-			</ItemGroup>
-		</When>
-		<Otherwise>
-			<ItemGroup>
-				<!-- Keep this until the project is effectively split in two to support the desktop build of WinUI -->
-				<PackageReference Include="Microsoft.ProjectReunion.WinUI">
-					<Version>0.5.6</Version>
-				</PackageReference>
-			</ItemGroup>
-		</Otherwise>
-	</Choose>
-	<PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">
-		<VisualStudioVersion>14.0</VisualStudioVersion>
-	</PropertyGroup>
-	<Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" />
-	<!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
```

Comment:
You should not need to change this file.

Reply:
I did not realize I did...